docs/adding_objectives.rst

```diff
@@ -234,3 +248,30 @@ available for the ``loss_function`` kwarg are ``[None,"mean","min","max"]``, wit
 ``None`` meaning using the usual default objective cost, while ``"mean"`` takes the
 average of the raw objective values (before subtracting the target/bounds or
 normalization), ``"min"`` takes the minimum, and ``"max"`` takes the maximum.
+
+Reducing Memory Size of Objective Jacobian Calculation
```

Comment:
I'd recommend removing this here.

I think its useful but should
- probably be put somewhere more focused, like a performance tips page
- also be tested more to find out what combos of batched/blocked/chunksize work best

Reply:
Yes, this deserves to have its own page.